### PR TITLE
Link to an associated tutorial entry on singletons

### PR DESCRIPTION
**Pull request by cevantes**

### CODE REVIEW

File: tutorials/tour/classes.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ object Classes {
 }
 ```
 
-The program defines an executable application Classes in form of a top-level singleton object with a `main` method. The `main` method creates a new `Point` and stores it in value `pt`. Note that values defined with the `val` construct are different from variables defined with the `var` construct (see class `Point` above) in that they do not allow updates; i.e. the value is constant.
+The program defines an executable application Classes in form of a top-level [singleton object](singleton-objects.html) with a `main` method. The `main` method creates a new `Point` and stores it in value `pt`. Note that values defined with the `val` construct are different from variables defined with the `var` construct (see class `Point` above) in that they do not allow updates; i.e. the value is constant.
```
Comment (jarrodwb) [inline tutorials/tour/classes.md]: I prefer links without the extention. It gives a cleaner experience. 

```
[singleton object](singleton-objects)
```

Comment (cevantes) [inline tutorials/tour/classes.md]: One can probably do it in one pass with sed

Comment (jarrodwb) [inline tutorials/tour/classes.md]: For sure. It would still need different PR. :+1:

**Comment by heathermiller** [comment]
LGTM, thanks guys!

**Pull request merged**